- disable mouse pointer support: uhexen2 handles mouse pointer from its in_amiga.c
- disable runloop support: uhexen2 doesn't use it.

diff -urNp MiniGL~/Build_gcc/makefile MiniGL/Build_gcc/makefile
--- MiniGL~/Build_gcc/makefile	2017-07-20 03:01:00.000000000 +0300
+++ MiniGL/Build_gcc/makefile	2017-08-20 03:11:00.000000000 +0300
@@ -16,9 +16,10 @@ VPATH = ../src/ ../demos
 
 INCLUDE = -I../include
 
-DEBUG = -DNDEBUG
-
-CFLAGS = -m68060 -m68881 -D__M68K__ $(DEBUG) -ffast-math
+CPPFLAGS = -DNDEBUG
+# -DMOUSE_SUPPORT
+# -DRUNLOOP_SUPPORT
+CFLAGS = -m68060 -m68881 -D__M68K__ $(CPPFLAGS) -ffast-math
 #-fomit-frame-pointer
 CFLAGS+= -O3 -DINLINE=__inline
 #CFLAGS+= -g
@@ -45,7 +46,8 @@ LIBOBJ = context.o vertexbuffer.o draw.o
 	others.o matrix.o
 
 
-all: $(LIBDIR) $(LIBNAME) $(DEMOS)
+all: $(LIBDIR) $(LIBNAME)
+# $(DEMOS)
 
 install:
 	cp ../include/mgl/*.h /usr/os-include/mgl
diff -urNp MiniGL~/makefile.vbcc68k MiniGL/makefile.vbcc68k
--- MiniGL~/makefile.vbcc68k	2017-07-20 03:01:00.000000000 +0300
+++ MiniGL/makefile.vbcc68k	2017-08-20 03:11:00.000000000 +0300
@@ -18,9 +18,12 @@
 
 INCLUDE   = -Iinclude
 # -IVBCC_INCLUDES/vincludem68k
-DEBUG = -DNDEBUG
-CFLAGS    = -O1 -speed -c99 $(INCLUDE) -cpu=68060 -fpu=68060 $(DEBUG)
-OP_CFLAGS = -O2 -speed -c99 $(INCLUDE) -cpu=68060 -fpu=68060 $(DEBUG)
+CPPFLAGS  = -DNDEBUG
+# -DMOUSE_SUPPORT
+# -DRUNLOOP_SUPPORT
+CFLAGS    = -O1 -speed -c99 $(INCLUDE) -cpu=68060 -fpu=68060 $(CPPFLAGS)
+OP_CFLAGS = -O2 -speed -c99 $(INCLUDE) -cpu=68060 -fpu=68060 $(CPPFLAGS)
+
 
 LIBDIR = lib/vbcc
 LIBS = -L$(LIBDIR) -lm040 -lmgl -lextra -lauto
@@ -46,7 +49,8 @@ $(DIR)/aclip.o $(DIR)/vertexarray.o $(DI
 #moved kprintf.o to EXE objects
 #$(DIR)/kprintf.o
 
-all: $(LIBDIR) $(LIBNAME) $(DEMOS)
+all: $(LIBDIR) $(LIBNAME)
+# $(DEMOS)
 
 install:
 	copy include/mgl/#?.h vinclude68k:mgl
diff -urNp MiniGL~/src/context.c MiniGL/src/context.c
--- MiniGL~/src/context.c	2017-07-20 03:01:00.000000000 +0300
+++ MiniGL/src/context.c	2017-08-20 03:11:00.000000000 +0300
@@ -70,8 +70,6 @@ static GLboolean newGuardBand = GL_FALSE
 
 static struct TagItem tags[7];
 
-static UWORD *MousePointer = 0;
-
 static GLboolean sys_MaybeOpenVidLibs(void)
 {
 #ifdef __PPC__
@@ -110,6 +108,13 @@ static GLboolean sys_MaybeOpenVidLibs(vo
 	return GL_TRUE;
 }
 
+#ifndef MOUSE_SUPPORT
+#define vid_Pointer(x)       do {} while (0)
+#define vid_DeletePointer(x) do {} while (0)
+#else
+
+static UWORD *MousePointer = 0;
+
 static void vid_Pointer(struct Window *window)
 {
 	if (!MousePointer)
@@ -134,6 +139,7 @@ static void vid_DeletePointer(struct Win
 #endif
 	MousePointer = 0;
 }
+#endif /* MOUSE_SUPPORT */
 
 void GLScissor(GLcontext context, GLint x, GLint y, GLsizei width, GLsizei height)
 {
diff -urNp MiniGL~/src/others.c MiniGL/src/others.c
--- MiniGL~/src/others.c	2017-07-20 03:01:00.000000000 +0300
+++ MiniGL/src/others.c	2017-08-20 03:11:00.000000000 +0300
@@ -448,6 +448,7 @@ void GLReadPixels(GLcontext context, GLi
 	free(pixelrect);
 }
 
+#ifdef RUNLOOP_SUPPORT
 void MGLKeyFunc(GLcontext context, KeyHandlerFn k)
 {
 	context->KeyHandler = k;
@@ -531,5 +532,5 @@ void MGLMainLoop(GLcontext context)
 		}
 	} /* While running */
 }
-
+#endif /* RUNLOOP_SUPPORT */
 
